Addon.Episerver.EnvironmentSynchronizer/Synchronizers/SiteDefinitions/SiteDefinitionSynchronizer.cs

```diff
@@ -110,5 +125,52 @@ private SiteDefinition GetExistingSiteDefinition(IEnumerable<SiteDefinition> exi
 
             return siteDefinition;
         }
+
+        private void SetForceLogin(SiteDefinition site, EnvironmentSynchronizerSiteDefinition siteDefinitionToUpdate)
```

Comment:
I suggest renaming this method to RemoveAccessForEveryoneRole

Reply:
Renamed